joern-cli/frontends/kotlin2cpg/src/main/scala/io/joern/kotlin2cpg/Main.scala

```diff
@@ -145,7 +146,12 @@ object Main extends App {
 
         val typeInfoProvider = new DefaultTypeInfoProvider(environment)
         val cpg = new Kt2Cpg().createCpg(filesWithMeta, fileContentsAtPath, typeInfoProvider, Some(config.outputPath))
+        val hasAtLeastOneMethodNode = cpg.method.take(1).size > 0
         cpg.close()
+        if (!hasAtLeastOneMethodNode) {
+          println("Error: resulting CPG does not contain any METHOD nodes.")
+          System.exit(1)
```

Comment:
what if i scan an empty artifact, wouldn't that result in 0 method nodes and still be correct?